### ISSUE TITLE
locations

### ISSUE THREAD

**Issue by qianwangn** [kkhoot/PAA#6]
That parameter seems not used.

`    def compute_reg_loss(self, regression_targets, box_regression,
                         all_anchors, labels, locations, n_anchors, weights, N):
        if 'iou' in self.reg_loss_type:
            reg_loss = self.GIoULoss(box_regression,
                                     regression_targets,
                                     all_anchors,
                                     weight=weights)
        elif self.reg_loss_type == 'smoothl1':
            reg_loss = smooth_l1_loss(box_regression,
                                      regression_targets,
                                      beta=self.bbox_reg_beta,
                                      size_average=False,
                                      sum=False)
            if weights is not None:
                reg_loss *= weights
        else:
            raise NotImplementedError
        return reg_loss[labels > 0].view(-1)`

**Comment by kkhoot** [comment]
Hi, thanks for reporting. I should have used some linters before releasing the code. :( I will clean this up soon. Thanks!

**Comment by kkhoot** [comment]
closing this as it is fixed now. Thanks again!
